server/models/Transaction.js

```diff
@@ -690,6 +688,42 @@ function defineModel() {
     return { transaction, platformTipTransaction, platformTipDebtTransaction };
   };
 
+  Transaction.createHostFeeTransactions = async ({ transaction, host }) => {
```

Comment:
In `createPlatformTipTransactions`, we now pass individual params

```suggestion
  Transaction.createHostFeeTransactions = async (transaction, host) => {
```